cmd/medius/images/push.go

```diff
@@ -247,6 +238,74 @@ func (b *buildAndPublish) readArtifact(artifactReader http.ReadCloserWithChecksu
 	return file.Name(), nil
 }
 
+func (b *buildAndPublish) buildImages(entry *common.Entry) ([]v1.Image, []string, error) {
+	var images []v1.Image
+	var artifacts []string
+
+	rebuildNeeded, err := b.rebuildNeeded(entry)
+	if err != nil {
+		return nil, nil, err
+	}
+	if !rebuildNeeded && !b.Options.PublishImagesOptions.ForceBuild {
+		b.Log.Info("Nothing to do.")
+		return nil, nil, nil
+	}
+
+	for i := range entry.Artifacts {
```

Comment:
use for `_, artifact := range entry.Artifacts {`
The code will be better readable

Reply:
Thanks. I'd prefer to leave it as `entry.Artifacts[i]`.